README.md

```diff
@@ -1,2 +1,45 @@
-# Hello-World
-my new repository
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello-World
+
+
+## Built With
+
+- HTML and CSS
+- VSCODE
+- GIT
+
+ 
+
+  
+
+
+
+## Authors
+
+👤 **Alusine Jalloh**
+
+- GitHub: [@githubhandle](https://github.com/Alusp)
+- LinkedIn: [LinkedIn](linkedin.com/in/alusine-jalloh-873951209)
```

Comment:
- [ **OPTIONAL** ] **RECOMENDED:** Good job by adding social links as the author. But I think it would be better if you replace text inside [...] square brackets with your usernames both for GitHub and Linkedin(e.g: [@Your_GItHub_username] instead of [@githubhandle], and the same with Linkedin)